manifests/download.pp

```diff
@@ -0,0 +1,64 @@
+# == Definition: archive::download
```

Comment:
similar to how the new parameters in the type are marked `(this parameter is for camptocamp/archive compatibility)`